### ISSUE TITLE
Package editor: Line properties

### ISSUE THREAD

**Issue by greenscreenflicker** [carrotIndustries/horizon#33]
It would be handy to have a start and a end position directly modifyable using a text entry.
![grafik](https://user-images.githubusercontent.com/17407241/34363064-ef7a11ec-ea78-11e7-86bc-b1d4f5825ce1.png)
Second, it would be nice to have a default line width.

See the pic for a rough scetch.

**Comment by atoav** [comment]
I agree to the ability to set default line widths (or maybe just remembering the last selected would work).

For the Positions I think `Enter Datum` could do what you want? 
![2018-02-01 12_13_28-enter datum](https://user-images.githubusercontent.com/6620266/35675739-7a464d30-0749-11e8-89d8-7c84d44faf23.png)
Having it in the sidebar makes probably also sense (there is enough space, you could display the values of all the junctions at once, etc)

**Comment by carrotIndustries** [comment]
> I agree to the ability to set default line widths (or maybe just remembering the last selected would work).

Currently Tools can't store any state between invocations, but being able to do so is on my todo list as well. Being able to set a default line width would be a byproduct of saving state.
